ncm-metaconfig/src/main/perl/metaconfig.pm

```diff
@@ -44,16 +42,6 @@ sub prepare_action
         }
     }
 
-    if ($srv->{daemon}) {
```

Comment:
ah yes, except in our case, we don't releases for lots of things, esp at this moment. after a while it converges, but not now :stuck_out_tongue: 
if we really start to rely on the coupling, we need to enforce this (or at least make it possible to enforce it).